src/Form/Model/RecoveryResetPassword.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+namespace App\Form\Model;
+
+/**
+ * @author doobry <doobry@systemli.org>
+ */
+class RecoveryResetPassword
+{
+    /**
+     * @var string
+     */
+    public $email;
+
+    /**
+     * @var string
+     */
+    public $recoveryToken;
```

Comment:
When you use getter/setter (i prefer that) then you should use only private properties.

Reply:
Agreed, fixed now.